DESCRIPTION

```diff
@@ -1,14 +1,19 @@
-Package: admiral.test
+Package: cdiscdata 
 Type: Package
-Title: Test Data for the {admiral} Package
+Title: Test Data from the CDISC Pilot Package
 Version: 0.1.0
 Authors@R: c(
   person("Thomas", "Neitmann", email = "thomas.neitmann@roche.com", role = c("aut", "cre")),
+   person(given = "Christina",
+             family = "Fillmore",
+             role = c("aut"),
+             email = "christina.e.fillmore@gsk.com",
+             comment = c(ORCID = "0000-0003-0595-2302")),
```

Comment:
Can you indent this by **two** spaces rather than three to be consistent with the other lines? And also please align the other lines with the first one.